src/components/external/Sidebar.vue

```diff
@@ -73,62 +87,93 @@ export default {
+          "items":[
+            {
+              "label":"SIDEBAR.HOME",
+              "icon":"house",
+              "viewUrl":"/home/index",
+            }
+          ]
+        },
+        {
+          "type":"category",
+          "label":"SIDEBAR.SYSTEMS",
+          "items":[
+            {
+              "label":"SIDEBAR.PUSH",
+              "icon":"hierarchy",
+              "viewUrl":"/systems/push",
+            },
+            {
+              "label":"SIDEBAR.BH",
+              "icon":"science-fiction-robot",
+              "viewUrl":"/systems/bothub",
+            },
+            {
+              "label":"SIDEBAR.RC",
+              "icon":"messaging-we-chat",
+              "viewUrl":"/systems/rocketchat",
             }
-          }),
-        };
-      });
+          ]
+        },
+        {
+          "type":"category",
+          "label":"SIDEBAR.PROFILE",
+          "items":[
+            {
+              "label":"SIDEBAR.ACCOUNT",
+              "icon":"single-neutral",
+              "viewUrl":"/account/edit",
+            },
+            {
+              "label":"SIDEBAR.PROJECT",
+              "icon":"folder",
+              "viewUrl":"/project/index",
+            },
+            {
+              "label":"SIDEBAR.HELP",
+              "icon":"question-circle",
+              "viewUrl":"/help/index",
+            }
+          ]
+        }
+      ].map(item => ({
+        ...item,
+        label: this.$t(item.label),
+        items: item.items.map(route => {
+          const active = this.$route.path.startsWith(route.viewUrl);
+
+          return {
+            ...route,
+            label: this.$t(route.label),
+            active,
+            icon: icons[route.icon][active ? 0 : 1],
+            click: () => {
+              this.$router.push(route.viewUrl);
+            },
+          };
+        })
+      }));
     },
 
     isToContract() {
       return [
         '/systems/push',
         '/systems/bothub',
         '/systems/rocketchat',
+      ].some((href) => this.$route.path.startsWith(href))
+        || [
         '/project',
-      ].some((href) => this.current.startsWith(href));
+      ].some((href) => this.$route.path === href);
     }
   },
   methods: {
-    async changeLanguage(language) {
-      if (language === window.Luigi.i18n().getCurrentLocale()) {
-        return false;
-      }
-
-      const languages = {
-        'en': 'en-us',
-        'pt-br': 'pt-br',
-      };
-
-      try {
-        const token = window.parent.Luigi.auth().store.getAuthData().accessToken;
-
-        await account.updateProfileLanguage({
-          headers: {
-            Authorization: `Bearer ${token}`,
-          },
-          language: languages[language],
-        });
-      } catch (error) {
-        console.log(error);
-      } finally {
-        window.Luigi.i18n().setCurrentLocale(language);
-      }
+    changeLanguage(language) {
+      this.$root.$emit('change language', language);
```

Comment:
Perhaps it would be better change emit name to change-language without space between